packages/permissionless/accounts/etherspot/utils/signTypedData.ts

```diff
@@ -0,0 +1,68 @@
+import {
+    type Account,
+    type Chain,
+    type Client,
+    type LocalAccount,
+    type SignTypedDataParameters,
+    type SignTypedDataReturnType,
+    type Transport,
+    type TypedData,
+    getTypesForEIP712Domain,
+    hashTypedData,
+    publicActions,
+    validateTypedData
+} from "viem"
+
+import {
+    signMessage as _signMessage,
```

Comment:
since there is no other `signMessage` we don't need to rename it.

Reply:
Changes done.